app/src/main/res/layout/activity_home.xml

```diff
@@ -6,24 +6,25 @@
     android:layout_height="match_parent"
     tools:context=".Home.Home">
 
-    <TextView
-        android:id="@+id/home_welcome_msg"
+    <androidx.emoji.widget.EmojiTextView
+        android:id="@+id/home_emoji_code"
```

Comment:
Not necessarily wrong for this PR, but we might want to put all the ids in a dedicated resource file.

Reply:
Ok, I didn't know we could do this. Will add an Issue later